master/internal/command/command.go

```diff
@@ -375,7 +372,11 @@ func (c *command) setPriority(ctx *actor.Context, priority int, forward bool) er
 		return fmt.Errorf("setting priority for job type %s in kubernetes is not supported",
 			c.jobType)
 	}
-	c.Config.Resources.Priority = &priority
+
+	currPriority := c.Config.Resources.Priority
+	if currPriority != nil && *currPriority != priority {
```

Comment:
do we ever need to set priority if the currPriority is nil? or do we expect Priority to always have a default value whenever this function called

also nit: `cur` vs `curr` feel free to ignore.

Reply:
ahh good catch